api/main.go

```diff
@@ -332,24 +343,14 @@ func main() {
 
 		query := []bson.M{
 			{
-				"$lookup": bson.M{
-					"from":         "connected_devices",
-					"localField":   "uid",
-					"foreignField": "uid",
-					"as":           "online",
-				},
+				"$lookup": device_query["lookup"],
 			},
 			{
-				"$lookup": bson.M{
-					"from":         "users",
-					"localField":   "tenant_id",
-					"foreignField": "tenant_id",
-					"as":           "namespace",
-				},
+				"$lookup": user_query["lookup"],
 			},
 			{
-				"$addFields": bson.M{
-					"online":    bson.M{"$anyElementTrue": []interface{}{"$online"}},
+				"$addFields": {
```

Comment:
I'm sure that is not valid code because type is missing in the composite literal